src/lib/nvme_access.c

```diff
@@ -44,7 +44,7 @@ static int send_recv_nvme_pt_ioctl(int fd, int send, uint8_t proto_id,
 	return status;
 }
 
-static int opal_send(int fd, uint16_t com_id, uint8_t *buf, int buf_len)
```

Comment:
I don't know what happen here, but I will remove the function name with secp at the end and passing in protocol id in instead.